static/src/stylesheets/module/atoms/_quiz.scss

```diff
@@ -0,0 +1,157 @@
+
+$accessibility-outline: #ffbf47;
```

Comment:
Even if it's totally unrelated? It seems like a tenuous connection. I think a well-labelled var is adequate, when the use case is specific and not generalised.